### PR TITLE
Release 2.7.0 - KP Deployment Changes

### PR DESCRIPTION
**Pull request by maheshkumargangula**
1. ItemSet API endpoint via knowledge-mw-service.
2. print-service deployment script.
3. content-service - module name (path) change in configuration.

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -102,7 +103,22 @@ premium_consumer_rate_limits:
...
+  - name: ItemSetAPIs
+    request_path: "{{ itemset_prefix }}/v1"
+    upstream_url: "{{ knowledge_mw_service_url }}/itemset/v1"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: contentAdmin
```
[the remark this answers is not in the document]
Comment (maheshkumargangula) [reply ansible/roles/kong-api/defaults/main.yml]: Done. Please validate.

File: ansible/roles/stack-sunbird/tasks/content-service.yml [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
+  shell: "docker service create --with-registry-auth --replicas {{ learner_replicas }} -p 9002:9000  --name content-service --hostname content-service --reserve-memory {{ learner_reservation_memory }} --limit-memory {{ learner_limit_memory }} --limit-cpu {{ learner_limit_cpu }} --reserve-cpu {{ learner_reservation_cpu }} --health-cmd 'wget -qO- content-service:9000/health || exit 1' --health-timeout 3s --health-retries 3  --network application_default --config source=content-service.conf,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/application.conf,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/tasks/content-service.yml]: @maheshkumargangula Using learner service variables is not accepted!

Comment (maheshkumargangula) [reply ansible/roles/stack-sunbird/tasks/content-service.yml]: @keshavprasadms - Done. Please validate.

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -102,7 +103,22 @@ premium_consumer_rate_limits:
...
+  - name: ItemSetAPIs
+    request_path: "{{ itemset_prefix }}/v1"
+    upstream_url: "{{ knowledge_mw_service_url }}/itemset/v1"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: "{{ updateItemSet_ACL | default(['contentUpdate']) }}"
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml]: add the variable as ItemSetAPIs_ACL instead of updateItemSet_ACL.

Comment (maheshkumargangula) [reply ansible/roles/kong-api/defaults/main.yml]: @SMY-ALTAMASH - Done. Please check.

**Comment by SMYALTAMASH** [comment]
This PR is now good to go.

**Pull request merged**